### DISCUSSION TITLE
WYSIWYG Editor

### DISCUSSION THREAD

**Discussion by abelardolg** [vuetifyjs/vuetify#12920] [Ideas]
Hi there,

Do you plan to add a WYSIWYG editor in the next release...or in the current one?

Best regards.

**Comment by Thor-x86** [comment]
I don't think they will add it any time soon because it will simply bloat the repo. Why not combine with the existing one like [CKEditor](https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html)?

**Comment by abelardolg** [comment]
Hi there,
I added a Summernote Vue version.

Thanks. Best regards.
